lib/services/screen-builder-service.ts

```diff
@@ -5,18 +5,24 @@ import * as path from "path";
 import * as util from "util";
 
 export class ScreenBuilderService implements IScreenBuilderService {
+	private static SCREEN_BUILDER_SPECIFIC_FILES = [".yo-rc.json", ".app.json", "app.js"];
+	private static UPGRADE_ERROR_MESSAGE = 'Your app has been build with an obsolete version of Screen Builder. Please, migrate it to latest version and retry.';
```

Comment:
Does the suggested migration work with the upgraded command or should users create a new SB project and move their custom code and code resources there? Make the migrate part more specific. E.g. "You need to create a new Screen Builder project and transfer custom code and resources to the newly created project."